<launch>
    <arg name="base_frame" default="base_footprint" />
    <arg name="odom_frame" default="odom" />
    <arg name="odom_topic_name" default="odom" />
    <arg name="pub_tf" default="false" />

    <node name="scout_base_node" pkg="scout_base" type="scout_base_node" output="screen" required="true">
        <param name="odom_frame" type="string" value="$(arg odom_frame)" />
        <param name="base_frame" type="string" value="$(arg base_frame)" />
        <param name="odom_topic_name" type="string" value="$(arg odom_topic_name)" />
        <param name="pub_tf" type="bool" value="$(arg pub_tf)" />
    </node>

</launch>
